_includes/known-issues.html

```diff
@@ -0,0 +1,33 @@
+{% assign pageName = page.title | downcase %}
+{% assign pageSlug = pageName | slugify %}
+{% assign gitHubLabel = page.gitHub_label_url | default: page.title | replace: ' ', '+' %}
+{% assign issues = site.data.issues[pageSlug] %}
+ 
+<h2 id="known-issues" class="site-component-section-title">Known issues</h2>
+
+<p>
+  All known issues for {{ page.title | downcase }} are tracked in 
+  <a href="https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Package%3A+{{ gitHubLabel }}%22">GitHub</a>.
+
+  {%- if issues.items -%}
+    &nbsp;Notable usability issues are tracked in the following table: </p>
```

Comment:
```suggestion
    &nbsp;Notable usability issues are tracked below: </p>
```

Reply:
It's no longer a table

Reply:
I'd love to get the passive voice out of this section as well, but it's not critical at the moment